README.md

```diff
@@ -104,46 +104,50 @@ source_profile = build-skeleton-packer
+| build\_region | The region in which to retrieve the base AMI from and build the new AMI. | `string` | `"us-east-1"` | no |
+| build\_region\_kms | The ID or ARN of the KMS key to use for AMI encryption. | `string` | `"alias/cool-amis"` | no |
+| is\_prerelease | The pre-release status to use for the tags applied to the created AMI. | `string` | `"false"` | no |
+| region\_kms\_keys | A map of regions to copy the created AMI to and the KMS keys to use for encryption in that region. The keys for this map must match the values provided to the aws\_regions variable. Example: {"us-east-1": "alias/example-kms"} | `map(string)` | `{}` | no |
+| release\_tag | The GitHub release tag to use for the tags applied to the created AMI. | `string` | `""` | no |
+| release\_url | The GitHub release URL to use for the tags applied to the created AMI. | `string` | `""` | no |
+| skip\_create\_ami | Indicate if Packer should not create the AMI. | `bool` | `false` | no |
+
+Changing these defaults can be done through a `.pkrvars.hcl` file:
+
+```hcl
+build_region = "us-east-2"
+build_region_kms = "alias/example-kms"
+is_prerelease = "true"
+```
 
 Here is an example of how to kick off a pre-release build:
 
 ```console
 pip install --requirement requirements-dev.txt
 ansible-galaxy install --force --force-with-deps --role-file src/requirements.yml
-export BUILD_REGION="us-east-1"
-export BUILD_REGION_KMS="alias/cool-amis"
-export GITHUB_RELEASE_TAG=$(./bump_version.sh show)
-AWS_PROFILE=cool-images-ec2amicreate-skeleton-packer packer build --timestamp-ui src/packer.json
+AWS_PROFILE=cool-images-ec2amicreate-skeleton-packer packer build --timestamp-ui -var release_tag=$(./bump_version.sh show) -var is_prerelease=true src/packer.pkr.hcl
 ```
 
 If you are satisfied with your pre-release image, you can easily create a release
 that deploys to all regions by adding additional regions to the packer configuration.
-This can be done with the `patch_packer_config.py` helper script.
-Echo in a comma-separated regions:kms_keys list to `patch_packer_config.py`
-and rerunning packer:
-
-```console
-echo "us-east-2:alias/cool-amis,us-west-1:alias/cool-amis,\
-us-west-2:alias/cool-amis" | ./patch_packer_config.py src/packer.json
-AWS_PROFILE=cool-images-ec2amicreate-skeleton-packer packer build --timestamp-ui src/packer.json
+This can be done by using a `.pkrvars.hcl` for example with `release.pkrvars.hcl`:
+
+```hcl
+ami_regions = ["us-east-2", "us-west-1", "us-west-2"]
+region_kms_keys = {
+  "us-east-2": "alias/cool-amis",
+  "us-west-1": "alias/cool-amis",
+  "us-west-2": "alias/cool-amis",
+}
 ```
 
-See the patcher script's help for more information about its options and
-inner workings:
-
 ```console
-./patch_packer_config.py --help
+AWS_PROFILE=cool-images-ec2amicreate-skeleton-packer packer build --timestamp-ui -var-file release.pkrvars.hcl src/packer.pkr.hcl
```

Comment:
Did you verify that if you follow these updated instructions, you can successfully build an AMI in one region and have it copied to one or more other regions?

Reply:
I have used these directions to create an AMI and have it copy to an additional region.